pyre/extra_tests/snippets/codecs_decoder_buffer_inputs.py

```
@@ -31,6 +35,30 @@
+region.close()
+
+for exporter in (
+    array.array("b", b"abc"),
+    ctypes.create_string_buffer(b"abc", 3),
+):
+    for decode in DECODERS:
+        assert decode(exporter) == ("abc", 3), (decode, exporter)
+
+
+class Exporter:
+    def __buffer__(self, flags):
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__buffer__`

(ANN204)